lang/dart.md

```diff
@@ -0,0 +1,206 @@
+- **[Rikulo Stream](https://github.com/rikulo/stream)** - Lightweight web server with request routing, filtering, template engine, WebSocket, MVC design pattern, and file-based static resources.
+- **[Alfred]( https://github.com/rknell/alfred)** - Minimal ExpressJS-inspired server with routing and middlware.
+ 
+ 
+## Why to use Dart?
+- Dart is really flexible
+- Dart embraced the open source ecosystem
+- Dart is pretty easy to learn
+- Dart is a good first language
+- Dart has great tooling support
+- Dart is a robust language
+- Dart ensures productivity
+- Dart is used by Flutter
+- Dart is backed by a tech giant
+- Dart solved a lot of problems, efficiently
+
+## Tools You Will Need
+For setting up environment for performing java codes you need with a minimum of 2 GB of RAM.
+You will also need the following softwares :
+- Linux, macOS or Windows xp/7/8/10 operating system
+- Dart SDK
+- Text Editors
+
+
+## Setting up Dart development environment
+**Executing Script Online with DartPad:**
+
+You may test your scripts online by using the online editor at https://dartpad.dartlang.org/. The Dart Editor executes the script and displays both HTML as well as console output. The online editor is shipped with a set of preset code samples.
+
+
+
+**Dart Installation:**
+ 
+ 
+**Install Dart on Windows:**
+In this tutorial, we will provide a step by step process to get Dart installed on Windows.
+
+Dart SDK comes as a pre-compiled version. So, download and extracting is all you need to do.
+
+**Step 1:** Download Dart SDK
+Go to Dart SDK archive page.
+
+The URL is https://dart.dev/tools/sdk/archive.
+
+Download Dart SDK
+Click on the Dart SDK link. The download will start immediately and a zip file will be downloaded.
+
+**Step 2:** Extract zip file
+Extract the contents of Dart SDK zip file. The contents of the folder would be as shown below.
+
+**Step 3:**
+Add Dart Path to PATH Environment Variable
+As of now, you can run dart command only from bin folder of dart sdk. To run dart command from anywhere in your file system, add dart bin path to PATH environment variable.
+
+Open Environment Variables. Under System variables, click on Path and click Edit button.
+
+Edit environment variable window appears. Click on New and paste the dart sdk bin path as shown below.
+
+**Step 4:** Restart Command Prompt
+Close the existing command prompt window and open a new command prompt.
+
+Just run the dart command:  `dart --version`
+
+
+**Install the Dart SDK on Linux:**
+
+
+The steps of Dart installation on Linux is given below.
+
+Before installing the Dart, if you are Debian/Ubuntu on AMD64(64-bit Intel) in your local machine, you can install the Dart through one of the following options.
+
+Install using apt-get
+Install a Debian package
+Installation using apt-get
+
+**Step -1:**
+
+ Type the following commands for a one-time setup.
+
+$sudo apt-get update  
```

Comment:
Surround code blocks with 3 backticks to render them as code blocks. Also, specify the syntax to allow us to set proper syntax highlighting.

Example code block:
<pre></code>```bash
sudo apt-get update
```</code></pre>

ℹ️ **Note:** Also, do not specify any elevation marks before the command as we'll be implementing a copy button in the future. Plus it's really inconvenient to copy the command that way... Feel free to contact me on Discord at `Vasanth Srivatsa#2000` on the [DEVSTR😊NS' Discord server](https://discord.gg/52VMn9edNH).